src/event.c

```diff
@@ -45,7 +45,7 @@ void event_free(struct event_loop *loop)
 
 int event_register(struct event_loop *loop, struct connection *c, int mask)
 {
-    struct epoll_event event;
+    struct epoll_event event = {0, {0}};
```

Comment:
Wired. valgrind in travis ci is also 3.10.1 and it has no uninitialized errors. https://travis-ci.org/eleme/corvus/builds/129413867#L356